eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -13,7 +13,7 @@ parameters:
   PRTitle: not-specified
   PRBody: ''
```

Comment:
Not sure if this change is worth flushing through eng/common but I guess if you are doing the sparse-checkout then it can come along for the ride.

Reply:
Yeah it's getting lumped in with the pre-existing display name changes. Figured it was easier to just let the whitespace fix tag along (rather than me unstaging this file every time I open it and my editor auto-deletes whitespace).